public/manifest.json

```diff
@@ -0,0 +1,25 @@
+{
+  "short_name": "React App",
```

Comment:
- Looks like you checked in the React logos (public/logo*)

Also, you should update this to be production values, this manifest gets used by Android and friends.